doc/api/assert.md

```diff
@@ -257,18 +257,23 @@ property set equal to the value of the `message` parameter. If the `message`
 parameter is undefined, a default error message is assigned.
 
 ## assert.fail(message)
-## assert.fail(actual, expected, message, operator)
+## assert.fail(actual, expected[, message[, operator[, stackStartFunction]]])
```

Comment:
Have a look at the examples below :smile: 

Using the operator requires to not use the message and this could not be fixed with any arguments switching as both should be strings. That's the only special case as far as I see it.